### PR TITLE
Switched the default logging to loguru.

### PR DESCRIPTION
**Pull request by ognjen-j**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from setuptools import setup
 
-required = ["confluent-kafka[avro]>=1.0", "pyaml>=3.13", "avro-python3"]
+required = ["confluent-kafka[avro]>=1.0", "pyaml>=3.13", "avro-python3", "loguru>=^0.4.0", "loguru"]
```
Comment (garrettthomaskth) [inline setup.py:3]: loguru is included twice

File: .travis.yml [hunk 2]
```diff
@@ -40,17 +40,17 @@ matrix:
     python: 3.7
     before_install: []
     env:
-    - TEST_CMD="flake8 esque/"
```
Comment (garrettthomaskth) [inline .travis.yml:43]: haha

File: setup.py (hunk 1 above)
Comment (ognjen-j) [reply setup.py]: PyCharm did that. Fixed it.

File: src/pyconnect/pyconnectsink.py [hunk 3]
```diff
@@ -377,8 +378,20 @@ def _commit(self) -> None:
...
+            while attempt_count <= max_attempts:
+                try:
+                    logger.info(f"Committing offsets: {offsets}")
+                    self._consumer.commit(offsets=offsets, asynchronous=False)
+                except KafkaException as ke:
+                    logger.error(
+                        f"Kafka exception occurred while comitting offsets (attempt {attempt_count}): {str(ke)}"
+                    )
+                    if attempt_count == max_attempts:
+                        exit(1)
```
Comment (swenzel) [inline src/pyconnect/pyconnectsink.py:392]:
```suggestion
                        raise
```
PyConnect has handlers for cleaning up after itself, we should give it a chance to do so.

File: src/pyconnect/pyconnectsink.py [hunk 4]
```diff
@@ -377,8 +378,20 @@ def _commit(self) -> None:
...
-            logger.info(f"Committing offsets: {offsets}")
-            self._consumer.commit(offsets=offsets, asynchronous=False)
+            max_attempts: int = 2
+            attempt_count: int = 1
+            while attempt_count <= max_attempts:
+                try:
+                    logger.info(f"Committing offsets: {offsets}")
+                    self._consumer.commit(offsets=offsets, asynchronous=False)
+                except KafkaException as ke:
+                    logger.error(
```
Comment (swenzel) [inline src/pyconnect/pyconnectsink.py:388]:
```suggestion
                    logger.warn(
```
Since we're retrying, a warning should be sufficient.

**Pull request merged**